python/tvm/topi/adreno/utils.py

```diff
@@ -571,6 +572,46 @@ def get_texture_storage(shape):
         return "global.texture-weight"
 
 
+# global texture variants has the same memory info
+gt_unit_bits = 16
```

Comment:
it's better to create function calling tvm.ir.make_node once and put all stuff inside this function instead of creation the global variables

Reply:
Remade